docs/Developer-Guide.md

```diff
@@ -564,10 +577,10 @@ an additional `coreutils` package.
 
 For example using CentOS:
 
-```
-$ cd $GOPATH/src/github.com/kata-containers/kata-containers/tools/osbuilder/rootfs-builder
-$ export ROOTFS_DIR=${GOPATH}/src/github.com/kata-containers/kata-containers/tools/osbuilder/rootfs-builder/rootfs
-$ script -fec 'sudo -E GOPATH=$GOPATH USE_DOCKER=true EXTRA_PKGS="bash coreutils" ./rootfs.sh centos'
+```bash
+$ pushd kata-containers/tools/osbuilder/rootfs-builder
+$ export ROOTFS_DIR="$(realpath ./rootfs)"
```

Comment:
There are double quotes around $(...) while getting the value of ROOTFS_DIR in above line and line# 314 but not in line# 239.

Reply:
Thank you for pointing that out. I updated the Readme to consistently use quotation marks since this is more stable.